aerogear/keycloak-operator
# [WIP] operator install

Placeholder as I dont have access to aerogear docker org
pushed it here https://hub.docker.com/r/witmicko/keycloak-operator/

### Current status
- provisioning without errors, waits for sso template.


## Verification notes
prerequisits:
- `oc cluster up --service-catalog=true`
- checkout https://github.com/integr8ly/installation/pull/19
- cd into `/integr8ly/installation/evals`
- comment out lines 5-9 in `/integr8ly/installation/evals/roles/rhsso-operator/tasks/main.yml`
- run `ansible-playbook -i inventories/hosts playbooks/rhsso-operator.yml` this will add rhsso template to catalog, hard reload catalog to see change, system dependent - it can take up to 10 minutes to show up

## verifcation notes
- checkout this PR
- `make install` 
- `make run`
- in another tab open this PR checkout and run
  * `oc create -f keycloak.json -n=rhsso` to provision
  * `oc delete keycloak example -n=rhsso` to deprovision
